framework/generated/khronos_generators/khronos_base_generator.py

```diff
@@ -1562,6 +1563,19 @@ def add_struct_members(self, name, value_info):
         self.all_struct_members[name] = value_info
         self.feature_struct_members[name] = value_info
 
+    def get_or_create_structs_with_null_pnexts(self):
+        """Returns the list of structs that must have null pNext pointers."""
+        if self.structs_with_null_pnexts is None:
+            self.structs_with_null_pnexts = self.struct_names.copy()
+            # The keys in validextensionstructs are base structures that can be extended by structures found in the values.
+            for struct_extended in self.registry.validextensionstructs.keys():
```

Comment:
I *think* we still want to discard the extension structs from `validextensionstructs.values()` (in addition to the base structs) because extension structs appear in pNext chains and therefore won't have spec mandated NULL pNext values.